src/main/java/seedu/address/logic/commands/client/AddClientCommand.java

```diff
@@ -20,51 +19,47 @@ public class AddClientCommand extends ClientCommand {
 
     public static final String COMMAND_FLAG = "-a";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a client to the address book. "
+    public static final String MESSAGE_ADD_CLIENT_USAGE = COMMAND_WORD + ": Adds a client to the address book. "
             + "Parameters: "
-            + PREFIX_NAME + "NAME "
-            + PREFIX_PHONE + "PHONE "
-            + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
-            + "[" + PREFIX_TAG + "TAG]...\n"
+            + PREFIX_CLIENT_NAME + "NAME "
+            + PREFIX_CLIENT_PHONE + "PHONE "
+            + PREFIX_CLIENT_EMAIL + "EMAIL "
+            + PREFIX_PROJECT_ID + "PROJECT ID: "
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "John Doe "
-            + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_CLIENT_NAME + "John Doe "
+            + PREFIX_CLIENT_PHONE + "98765432 "
+            + PREFIX_CLIENT_EMAIL + "johnd@example.com "
+            + PREFIX_PROJECT_ID + "1";
 
     public static final String MESSAGE_SUCCESS = "New client added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This client already exists in the address book";
+    public static final String MESSAGE_DUPLICATE_CLIENT = "This client already exists in the address book";
 
-    private final Client toAdd;
+    private final Client toAddClient;
 
     /**
      * Creates an AddCommand to add the specified {@code Client}
      */
     public AddClientCommand(Client client) {
         requireNonNull(client);
-        toAdd = client;
+        toAddClient = client;
     }
 
-    // TODO: revise implementation
     @Override
     public CommandResult execute(Model model, Ui ui) throws CommandException {
         requireNonNull(model);
 
-        if (model.hasClient(toAdd)) {
-            throw new CommandException(MESSAGE_DUPLICATE_PERSON);
+        if (model.hasClient(toAddClient)) {
+            throw new CommandException(MESSAGE_DUPLICATE_CLIENT);
         }
 
-        model.addClient(toAdd);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+        model.addClient(toAddClient);
+        return new CommandResult(String.format(MESSAGE_SUCCESS, toAddClient));
```

Comment:
might want to add: 

ui.showClients()
model.updateFilteredClientList(PREDICATE_SHOW_ALL_CLIENTS);

so that when a client is updated, it'll display the client list